src/main/java/picard/sam/SamErrorMetric/CollectSamErrorMetrics.java

```diff
@@ -143,6 +172,14 @@
     @Argument(shortName = "L", doc = "Region(s) to limit analysis to. Supported formats are VCF or interval_list. Will intersect inputs if multiple are given. ", optional = true)
     public List<File> INTERVALS;
 
+    @Argument(shortName = "F", doc = "Filters for read outputs. Within each file, the first line defines the name of the filter. " +
+            "Each subsequent line represents one criterion. Each criterion is represented by 3 tab-separated fields: " +
+            "\"suffix(TAB)datatype(TAB)operator(TAB)value\", where datatype can be any of \"boolean, int\" and operator " +
+            "can be one of \"=, !=, <, <=, >, >=\" (depending on the datatype). These criteria are conjunctive, "+
+            "i.e. all criteria must be met for the read to be included in the output. Multiple files can be provided which are disjuncitve to each " +
+            "other (i.e. reads are included if they meet either of the filters).", optional = true)
+    public List<File> FILTERS;
```

Comment:
Can you have a test example where FILTERS is used? it's always good to see how the author envisions parameters being used...

Reply:
what's the argument for needing this new filtering DSL?
Also, what does the name of the filter mean? how does the name of the filter get used? 
finally, it isn't clear from the description what is being filtered...what is suffix?

Reply:
1. Some FILTER files are being parsed in ErrorReadFilterTest.java -> testFilterFileParsing. The FILTER argument isn't used directly, I can write an example test for that
1. The idea of the filtering is to being able to output reads that match certain criteria to being able to look at them manually afterwards which, as far as I understand it, is of particular interest for our long read investigations. Let me know if this feature shouldn't be included in master and we'll keep in a separate branch
1. The filter name determines the name of the output file created which lists reads that match that specific filter's criteria. The suffix corresponds to the stratifier name used for the criterion. I updated the description to reflect that.

Reply:
Seems like this should be in a separate tool. 
I would use the ENUM name of the stratification rather than the suffix, in the DSL.